ppr-ui/src/views/newMhrRegistration/YourHome.vue

```diff
@@ -2,11 +2,31 @@
   <div id="mhr-describe-your-home">
     <section id="mhr-make-model" class="mt-10">
       <h2>Manufacturer, Make, and Model</h2>
-      <p class="mt-2 mb-6">
-        Enter the Year of Manufacture (not the model year), Make, and Model of
-        the home.
+      <p :class="['mt-2', showHelp ? 'mb-3' : 'mb-6']" data-test-id="make-model-prompt">
+        {{ manufacturerMakeModelPrompt }}
       </p>
 
+      <SimpleHelpToggle
+        v-if="showHelp"
+        class="mb-7"
+        toggleButtonTitle="Need Help? Contact Us"
+        :defaultHideText="true"
```

Comment:
Do we need to specify that we use a default hide text explicitly? I mean, if it's a default text - it should be displayed as a fallback by default.

Reply:
Makes sense, ill change the other occurences to specify it as false.